Jbarget/oxymore
# So/erotic stories text

### What changes have you made?
replaced the placeholder text with formatted text for: 

- Bellybuttons (Spanish and English)
- Wings (English and Spanish) - Note. there is no italics formatting on the Spanish version confirmed by Aitor

Also, we no longer have the scroll on the text since we applied the `flex: auto`. Are we best to override the `flex:auto` in this case? I played around with height and max-height but didn't manage to find the solve here

### Which issue(s) does this PR fix?
Fixes #216 

### Screenshots (if there are design changes)


### How to test
Go to http://localhost:3000/projects/eroticstories and check that: 

- there are 2 formatted erotic stories available in english and spanish 
- there are no rogue `</br>`s or `<i>`s in the text
